.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -12,7 +12,6 @@
 import tzlocal
 import tempfile
 
```

Comment:
Probably want to get rid of the whitespace change here, see 

https://submitty.org/developer/testing/linting_static_analysis#php-linting
and
https://submitty.org/developer/testing/linting_static_analysis#php-linting